io.github.OkuBrowser.oku.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id" : "io.github.OkuBrowser.oku",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "oku",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--socket=cups",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=host",
```

Comment:
you don't need the first 2 if you have host

Reply:
but then maybe you don't need host.

Reply:
First two are redundant and have been removed. The browser needs full filesystem access.

Reply:
> The browser needs full filesystem access.

Why? Other browsers like Firefox does not need this.